packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -472,26 +479,28 @@ contract DeployImplementations is Script {
     // Deploy and initialize a proxied OPStackManager.
     function createOPSMContract(
         DeployImplementationsInput _dii,
-        DeployImplementationsOutput,
-        OPStackManager.Blueprints memory blueprints,
-        string memory release,
-        OPStackManager.ImplementationSetter[] memory setters
+        DeployImplementationsOutput _dio,
```

Comment:
Looks like we're following the convention everywhere so adding it to these functions too.